peds-protocol-questionnaires/peds-protocol/peds-protocol

```diff
@@ -1,7 +1,7 @@
 {
   "@context": "https://raw.githubusercontent.com/ReproNim/reproschema/main/releases/1.0.0/reproschema",
   "@type": "reproschema:Protocol",
-  "@id": "peds-protocol_combined",
+  "@id": "peds-protocol",
```

Comment:
so this ID for the questionnaire is the same as for the combined? since you cleaned up a lot of typos, this seems like it would be a good change

Reply:
this @id should be the same as the json filename. and they belong to different protocols, so it's okay they both use `peds-protocol` as id.